### PR TITLE
[android]Turn off fbgemm for android build, fixes libtorch android x86

### PR DESCRIPTION
**Pull request by IvanKobzarev**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#24057 [android]Turn off fbgemm for android build, fixes libtorch android x86**

### CODE REVIEW

File: scripts/build_android.sh [hunk 1]
```diff
@@ -89,6 +89,7 @@ CMAKE_ARGS+=("-DUSE_LMDB=OFF")
 CMAKE_ARGS+=("-DUSE_LEVELDB=OFF")
 CMAKE_ARGS+=("-DUSE_MPI=OFF")
 CMAKE_ARGS+=("-DUSE_OPENMP=OFF")
+CMAKE_ARGS+=("-DUSE_FBGEMM=OFF")
```
Comment (ljk53) [inline scripts/build_android.sh]: another option is to do this in master CMake directly? https://github.com/pytorch/pytorch/blob/master/CMakeLists.txt#L224
as we might need create other scripts for different type of mobile builds it probably makes sense to add this to a centralized place?

Comment (IvanKobzarev) [reply scripts/build_android.sh]: Thanks.
I tried to update this pull request, but accidentally opened a new one with updated commit

https://github.com/pytorch/pytorch/pull/24104

**Comment by ezyang** [comment]
@IvanKobzarev Something very strange happened to this PR. Do you still have the terminal history for your interaction with ghstack?

**Comment by IvanKobzarev** [comment]
Abandoning in favor of https://github.com/pytorch/pytorch/pull/25113

**Pull request closed without merging**